documentation/src/main/resources/pages/ditto/release_notes_200.md

```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
+| `- ditto-model-thingsearch`                  | `ditto-thingsearch/ditto-thingsearch-model` |
+| `- ditto-model-thingsearch-parser`           | `ditto-rql/ditto-rql-parser` |
+| `ditto-protocol-adapter`                     | `ditto-protocol` |
+| `ditto-signals`                              | `-` (was pom only) |
+| `- ditto-signals-base`                       | `ditto-model-base` |
+| `- ditto-signals-acks`                       | `-` (was pom only) |
+| `-- ditto-signals-acks-base`                 | `ditto-base-model` |
+| `-- ditto-signals-acks-things`               | `ditto-things-model` |
+| `- ditto-signals-announcements`              | `-` (was pom only) |
+| `-- ditto-signals-announcements-base`        | `ditto-base-model` |
+| `-- ditto-signals-announcements-policies`    | `ditto-policies-model` |
+| `- ditto-signals-commands`                   | `-` (was pom only) |
+| `-- ditto-signals-commands-base`             | `ditto-base/ditto-base-model` |
+| `-- ditto-signals-commands-cleanup`          | `-` (was internal API) |
+| `-- ditto-signals-commands-common`           | `-` (was internal API) |
+| `-- ditto-signals-commands-connectivity`     | `ditto-connectivity/ditto-connectivity-model` |
+| `-- ditto-signals-commands-devops`           | `ditto-devops/ditto-devops-model` |
+| `-- ditto-signals-commands-messages`         | `ditto-messages/ditto-messages-model` |
+| `-- ditto-signals-commands-namespaces`       | `ditto-base/ditto-base-model` |
+| `-- ditto-signals-commands-policies`         | `ditto-policies/ditto-policies-model` |
+| `-- ditto-signals-commands-things`           | `ditto-things/ditto-things-model` |
+| `-- ditto-signals-commands-thingsearch`      | `ditto-thingsearch/ditto-thingsearch-model` |
+| `- ditto-signals-events`                     | `-` (was pom only) |
+| `-- ditto-signals-events-base`               | `ditto-base-model` |
+| `-- ditto-signals-events-connectivity`       | `ditto-connectivity/ditto-connectivity-model` |
+| `-- ditto-signals-events-policies`           | `ditto-policies/ditto-policies-model` |
+| `-- ditto-signals-events-things`             | `ditto-things/ditto-things-model` |
+| `-- ditto-signals-events-thingsearch`        | `ditto-thingsearch/ditto-thingsearch-model` |
+
+#### Restructuring of Ditto's Java packages
+
+When updating from Ditto 1.x Java APIs (e.g. also when using the [Ditto Java client](#ditto-java-client)), the following
+packages were renamed:
+
+| Ditto 1.x package                                   | Ditto 2.x package      |
+| ---                                                 | ---                    |
+| `org.eclipse.ditto.model.base`                      | `org.eclipse.ditto.base.model` |
+| `org.eclipse.ditto.model.cleanup`                   | `-` (was internal API) |
+| `org.eclipse.ditto.model.connectivity`              | `org.eclipse.ditto.connectivity.model` |
+| `org.eclipse.ditto.model.devops`                    | `org.eclipse.ditto.devops.model` |
```

Comment:
ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API